builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
yeah and that case is covered by `autoplay` and lack of `controls`. `<amp-video autoplay loop>` would exactly do what you are describing.

What we are missing now is ability to autoplay with sound which would only work in desktop. I suggest not worrying about that case yet, if we need to have it, we can add support either using `autoplay=withsound` or `autoplay-with-sound`. More interesting (and more challenging) would be trying to carry user's intention of watching a video all the way from the platform in a way that browser would allow us to call `play()` on unmuted videos to get unmuted autoplay in mobile.